engine/installation/linux/ubuntu.md

```diff
@@ -71,7 +71,7 @@ Docker from the repository.
 2.  Add Docker's official GPG key:
 
     ```bash
-    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add
+    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add -
```

Comment:
If we need to use https, then the second option is better. Can you also change it for the Debian instructions, @artheus ? Mind the space between the `$` and the `curl` though.